src/plugins/intel_cpu/src/nodes/mkldnn_fullyconnected_node.cpp

```diff
@@ -108,44 +110,118 @@ void MKLDNNFullyConnectedNode::getSupportedDescriptors() {
         outputDataType = memory::data_type::bf16;
     }
 
-    const auto& inDims = getInputShapeAtPort(0).getStaticDims();
-    const auto& outDims = getOutputShapeAtPort(0).getStaticDims();
+    const auto& inShape = getInputShapeAtPort(DATA_ID);
+    inDummyDims = inShape.getDims();
```

Comment:
Renamed. I named it `inDummyDims` because I wanted to emphasize that this dims must be used only on load network stage not inference. Maybe we can name it `inPrimDims`? What do you think?